crates/sargon/src/wrapped_radix_engine_toolkit/high_level/manifest_building/manifest_delete_account.rs

```diff
@@ -0,0 +1,118 @@
+use crate::prelude::*;
+use radix_engine_interface::blueprints::account::{
+    AccountSetResourcePreferenceInput as ScryptoAccountSetResourcePreferenceInput,
+    ResourcePreference as ScryptoResourcePreference, ACCOUNT_SECURIFY_IDENT,
+    ACCOUNT_SET_RESOURCE_PREFERENCE_IDENT,
+};
+
+impl TransactionManifest {
+    pub fn delete_account(
+        account_address: &AccountAddress,
+        fee_payer_address: &AccountAddress,
+    ) -> Self {
+        let mut builder = ScryptoTransactionManifestBuilder::new();
+        let bucket_factory = BucketFactory::default();
+
+        // Lock fee
+        builder = builder.lock_standard_test_fee(fee_payer_address.scrypto());
```

Comment:
hmm I think we ALWAYS lock fee outside of a built manifest no?

Reply:
right, this will be added by Host apps as this manifest will be submitted as regular TX. Will remove